utils/local-engine/Operator/ExpandStep.cpp

```diff
@@ -59,21 +59,7 @@ DB::Block ExpandStep::buildOutputHeader(
     for (size_t i = 0; i < input_header.columns(); ++i)
     {
         const auto & old_col = input_header.getByPosition(i);
-        if (i < aggregating_expressions_columns_.size())
-        {
-            // do nothing with the aggregating columns.
-            cols.push_back(old_col);
-            continue;
-        }
-        if (old_col.type->isNullable())
-            cols.push_back(old_col);
-        else
-        {
-            auto null_map = DB::ColumnUInt8::create(0, 0);
-            auto null_col = DB::ColumnNullable::create(old_col.column, std::move(null_map));
-            auto null_type = std::make_shared<DB::DataTypeNullable>(old_col.type);
-            cols.push_back(DB::ColumnWithTypeAndName(null_col, null_type, old_col.name));
-        }
```

Comment:
The reason add null here is that , for `group by a,b roll up`, the column `a`, `b` after this `expand` node could be null.  we will have the following cases
|  a   | b  | explain |
|  ----  | ----  | ---- |
| original a's values  | original b's values | equal to group by a,b |
| original a's values  | NULL | equal to group by a |
| NULL | NULL | equal to group by empty keys |